packages/mobile/src/components/Feed/Interacts/Comments/CommentHead.tsx

```diff
@@ -35,7 +35,7 @@ export const CommentHead = memo(
         label: `${t('Block this user')}`,
         onPress: () =>
           openURL(
-            `${new DotYouClient({ identity, api: ApiType.Guest }).getRoot()}/owner/connections/${authorOdinId}/block`
+            `${new DotYouClient({ loggedInIdentity: identity, api: ApiType.Guest }).getRoot()}/owner/connections/${authorOdinId}/block`
```

Comment:
This will have to be the`hostIdentity`, the HostIdentity is the identity used to send requests to, and the loggedInIdentity can be a different one, especially in case of our first party apps. But for chat the hostIdentity will always need to be set. (I'll see if I can better enforce that on the type)